package.json

```
@@ -1,14 +1,14 @@
 {
-    "name": "minecraft-java-core",
-    "version": "3.10.4",
+    "name": "minecraft-java-core-azbetter",
+    "version": "3.11.0",
```

And I don't think you should be the one who bumps the deps
```suggestion
    "version": "3.10.4",
```